### PR TITLE
[Labs-715][GenAI] Continue and Update buttons pushed off screen

### PR DESCRIPTION
**Pull request by levadadenys**
### [Labs-715][GenAI] Continue and Update buttons pushed off screen

* Update  min width in order to prevent ui to be jumpy on when screen width is small

Fixed the issue by setting the `min-width` to every column and adding `overflow: auto`, so that in case the screen width is too small - user will be able to just scroll horizontally. 200px is a value got from visual comparison (screen collapses when columns width is smaller than 196.7px, we use x8 factor for design system, so used the closes x8 factor here, which is 200px )

Before:
scrolling-bug

After:

https://github.com/code-dot-org/code-dot-org/assets/22244040/3db6b0b1-4e2d-4856-8031-b63dd6de559d

## Links

- jira ticket: [LABS-715](https://codedotorg.atlassian.net/browse/LABS-715)

## PR Checklist:

- [ ] Tests provide adequate coverage
- [ ] Privacy and Security impacts have been assessed
- [ ] Code is well-commented
- [ ] New features are translatable or updates will not break translations
- [ ] Relevant documentation has been added or updated
- [ ] User impact is well-understood and desirable
- [ ] Pull Request is labeled appropriately
- [ ] Follow-up work items (including potential tech debt) are tracked and linked

### CODE REVIEW

**Review by thomasoniii — APPROVED** [review]
lgtm 🚀

File: apps/src/aichat/views/aichatView.module.scss [hunk 1]
```diff
@@ -19,15 +19,18 @@
   .instructionsArea {
     width: 30%;
     height: 100%;
+    min-width: 200px;
```
Comment (sanchitmalhotra126) [inline apps/src/aichat/views/aichatView.module.scss]: Mind adding a comment similar to your PR description around why this value was chosen? Optional - also, create a sass constant for this value?

**Review by sanchitmalhotra126 — APPROVED** [review]
Nice solution!

File: apps/src/aichat/views/aichatView.module.scss [hunk 2]
```diff
@@ -19,15 +19,18 @@
   .instructionsArea {
     width: 30%;
     height: 100%;
```
Comment (fisher-alice) [inline apps/src/aichat/views/aichatView.module.scss:27]: I wonder if we should also prevent buttons from disappearing if user resizes window vertically.

Comment (levadadenys) [reply apps/src/aichat/views/aichatView.module.scss:27]: It's also included in this PR. If user resizes window vertically and height is too small - scroll for y axis will appear

Comment (levadadenys) [reply apps/src/aichat/views/aichatView.module.scss:27]: https://github.com/code-dot-org/code-dot-org/assets/22244040/76bb45e2-94bd-482e-b0c5-bfba56522765

Comment (fisher-alice) [reply apps/src/aichat/views/aichatView.module.scss:27]: Nice! Thanks for the demo.

**Review by fisher-alice — APPROVED** [review]
🎉

**Pull request merged**